docs/topics/backend-details.rst

```diff
@@ -123,90 +28,42 @@ For docs see: https://docs.docker.com/
 FAQ
 ---
 
-Seeing a "Couldn't connect to Docker daemon..." error
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-If you see something like::
-
-  $ fig run zamboni ./manage.py dbshell
-  Couldn't connect to Docker daemon at http+unix://var/run/docker.sock - is
-  it running?
-
-  If it's at a non-standard location, specify the URL with the DOCKER_HOST
-  environment variable.
-
-It's likely you've not set the DOCKER_HOST env variable on OSX. If you run
-`boot2docker up` it will tell you what value it should be set to. Add this
-to your `.bashrc` or equivalent so it's set for all shells.
-
-Getting a "Couldn't start container" error
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-If you see something like the following on boot2docker/docker::
-
-  Cannot start container c44d451fcb58853bd9ef6d13ba4edf100817fce75bbfe7f9c814d68a708d82e3: setup
-  mount namespace bind mounts stat /Users/whatevar/git/marketplace-env/trees/spartacus: no such file or directory
-
-or something like this::
-
-  nginx_1 | nginx: [emerg] host not found in upstream "webpay_1:2601" in /etc/nginx/conf.d/marketplace.conf:2
-
-Then it's likely fig can't see the source code. Check that the root is correct,
-`mkt check` will do this for you.
-
-If you're on OSX this probably means the shared folders are not working for some reason.
-
-`fig build` fails on Linux saying it can't connect to the daemon
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-You need to add your user to the `docker` group and probably log out/in again to make sure you
-are there (run `groups` and make sure it says docker to verify)
-
-See http://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo
-
 How do I run migrations (Python projects)?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Here's the command (runs in a new instance)::
 
-  fig run --rm [image] schematic migrations/
+  docker-compose run --rm [image] schematic migrations/
 
 E.g (for zamboni)::
 
-  fig run --rm zamboni schematic migrations/
+  docker-compose run --rm zamboni schematic migrations/
 
 How do I run Python unit tests?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 This command will run the unittests in a new instance::
 
-  fig run --rm [image] python ./manage.py test --noinput -s --logging-clear-handlers
+  docker-compose run --rm [image] python ./manage.py test --noinput -s --logging-clear-handlers
 
 E.g. (for zamboni)::
 
-  fig run --rm zamboni python ./manage.py test --noinput -s --logging-clear-handlers
+  docker-compose run --rm zamboni python ./manage.py test --noinput -s --logging-clear-handlers
```

Comment:
Would it make sense to just get people used to just doing docker-compose exec ... or docker-utils bash. Because then it just a matter of saying "once you have got a prompt in the container, do everything as normal".

Thinking of docker as just a linux VM I hop into and not something special made life much easier for me.